services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentService.kt

```diff
@@ -157,6 +173,14 @@ class ApplicationAttachmentService(
     private fun sendAttachment(alluId: Int, attachment: ApplicationAttachmentEntity) {
         cableReportService.addAttachment(alluId, attachment.toAlluAttachment())
     }
+
+    private fun attachmentAmountExceeded(applicationId: Long): Boolean {
```

Comment:
The attachment isn't exceeded until there are at least 21 attachments, so the name is slightly inaccurate.

```suggestion
    private fun attachmentAmountReached(applicationId: Long): Boolean {
```

Reply:
Fixed.